src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -52,10 +52,13 @@ public Http3Connection(HttpMultiplexedConnectionContext context)
 
         _errorCodeFeature = context.ConnectionFeatures.GetRequiredFeature<IProtocolErrorCodeFeature>();
 
-        var httpLimits = context.ServiceContext.ServerOptions.Limits;
+        var serverOptions = context.ServiceContext.ServerOptions;
+        var httpLimits = serverOptions.Limits;
 
         _serverSettings.HeaderTableSize = (uint)httpLimits.Http3.HeaderTableSize;
         _serverSettings.MaxRequestHeaderFieldSectionSize = (uint)httpLimits.MaxRequestHeadersTotalSize;
+        _serverSettings.EnableWebTransport = 1;
+        _serverSettings.H3Datagram = 0; // we don't support this yet
```

Comment:
You need this enabled for the browser right?

Reply:
I do. I just wasn't sure if I should push it since we don't support it. I guess your comment means I should.